services/log-management.html.md.erb

```diff
@@ -11,6 +11,8 @@ Cloud Foundry aggregates logs for all instances of your apps as well as for requ
 
 If you want to persist more than the limited amount of logging information that Cloud Foundry can buffer, drain these logs to a log management service.
 
+<b>Feature:</b>  Since [CAPI release 1.141.0](https://github.com/cloudfoundry/capi-release/releases/tag/1.141.0) you've got also the opportunity to used mTLS inside your syslog drain. Please, follow the described process inside <a href="#step2">Step 2: Create and Bind a User-Provided Service Instance</a> to handle it and inject the corresponding credentials as <em>PEM</em> encoded <em>X.509</em> certificate.
```

Comment:
s/you've got also the opportunity to used mTLS/you can use mTLS

Reply:
I would prefer to stop after the link and describe the details below.

Reply:
Done